src/components/core/Heading/Heading.js

```diff
@@ -9,22 +10,24 @@ const Heading = styled(Box).attrs({
 	is: props => props.is,
 	m: 0,
 })`
-	font-family: ${props => props.theme.fontFamily.heading};
-	font-size: ${props => props.theme.fontSize[props.size]};
-	color: ${props => props.theme.color.base};
-	font-weight: normal;
+	${props =>
+		text({
+			lineHeight: 1.2,
+			fontFamily: 'heading',
+			fontSize: props.size,
+		})};
 `;
 
 Heading.propTypes = {
 	/** Custom component or HTML tag */
 	is: PropTypes.oneOfType([PropTypes.element, PropTypes.string]),
-	size: PropTypes.oneOf(['xxlarge', 'xlarge', 'large', 'base', 'small']),
+	size: PropTypes.oneOf(['xxl', 'xl', 'l', 'base']),
```

Comment:
Perhaps `lg` instead of `l`. Also `base` has semantics where others not. Consider `md`. Also why is there no small sizes?